bh_modules/pascalkeywords.py

```diff
@@ -0,0 +1,14 @@
+"""
+BracketHighlighter.
+
+Copyright (c) 2013 - 2015 Isaac Muse <isaacmuse@gmail.com>
+License: MIT
+"""
+
+
+def compare(name, first, second, bfr):
+    """Differentiate 'repeat..until' and 'unit..end.' from '*..end' brackets."""
+    brackets = (bfr[first.begin:first.end], bfr[second.begin:second.end])
+    return (brackets[0] == "repeat" and brackets[1] == "until") \
+        or (brackets[0] == "unit" and brackets[1] == "end.") \
```

Comment:
Luckily the continuations no longer matter as I've removed the `unit` support.  I'll keep that in mind for future Python work.